CMakeLists.txt

```diff
@@ -213,6 +213,7 @@ if(opencoarrays_aware_compiler)
   add_mpi_test(get_with_offset_1d 2 ${tests_root}/unit/send-get/get_with_offset_1d)
   add_mpi_test(whole_get_array 2 ${tests_root}/unit/send-get/whole_get_array)
   add_mpi_test(strided_get 2 ${tests_root}/unit/send-get/strided_get)
+  add_mpi_test(strided_sendget 4 ${tests_root}/unit/send-get/strided_sendget)
```

Comment:
Is there any harm in over subscribing?

I've found that often over subscribing can find race conditions and other parallel programming errors that might not otherwise show up.

For OpenMPI, the tests will run on the number of available cores, because current OpenMPI implementations are unreliable (at least on our virtualized testing infrastructure)